src/gluonts/torch/scaler.py

```diff
@@ -37,21 +36,12 @@ class MeanScaler(nn.Module):
         minimum possible scale that is used for any item.
     """
 
-    @validated()
-    def __init__(
-        self,
-        dim: int = -1,
-        keepdim: bool = True,
-        default_scale: float = 0.0,
-        minimum_scale: float = 1e-10,
-    ):
-        super().__init__()
-        self.dim = dim
-        self.keepdim = keepdim
-        self.register_buffer("minimum_scale", torch.tensor(minimum_scale))
-        self.register_buffer("default_scale", torch.tensor(default_scale))
-
-    def forward(
+    dim: int = -1
+    keepdim: bool = True
+    default_scale: float = 0.0
+    minimum_scale: float = 1e-10
```

Comment:
I think these need to be `torch.tensor`

Reply:
`torch.clamp` also deals with numbers; for the `default_scale`, isn't this some kind of "immutable" (mind the quotes) property of the object, so that replacing `torch.where` with `if` is not really harmful?